native/acter/src/api/message.rs

```diff
@@ -1270,10 +1261,7 @@ impl RoomMessage {
             event.origin_server_ts.get().into(),
             "m.room.name".to_string(),
         );
-        let body = match event.content.name {
-            Some(name) => format!("changed name to {name}"),
-            None => "changed name".to_string(),
-        };
+        let body = format!("changed name to {}", event.content.name);
```

Comment:
same as above.

Reply:
`event.content.name` is not an `Option<String>` anymore.